private/buf/cmd/buf/buf.go

```diff
@@ -317,14 +317,48 @@ func wrapError(err error) error {
 		connectCode := connectErr.Code()
 		switch {
 		case connectCode == connect.CodeUnauthenticated, isEmptyUnknownError(err):
-			if authErr, ok := bufconnect.AsAuthError(err); ok && authErr.TokenEnvKey() != "" {
+			loginCommand := "buf registry login"
+			authErr, ok := bufconnect.AsAuthError(err)
+			if !ok {
+				// This code should be unreachable.
+				return fmt.Errorf("Failure: you are not authenticated. "+
+					"Set the %[1]s environment varialbe or run %q, using a Buf API token as the password. "+
```

Comment:
Typo on variable

Reply:
fixed in [e40602f](https://github.com/bufbuild/buf/pull/3093/commits/e40602f11652d62e2a9c6c6f201b3dd8716be904)